### ISSUE TITLE
Provide serialization example in the docs for Database storage

### ISSUE THREAD

**Issue by feinstein** [JavaMoney/jsr354-ri#248]
Are there any samples on how to save `MonetaryAmount` on a database? I think this is very common use case, so the docs should include it.

**Comment by stokito** [comment]
https://github.com/JavaMoney/jsr354-ri/issues/185

You are welcome to send a PR  to userguides.
Actually the best guides are those which written by users when they had a question.

**Comment by feinstein** [comment]
Good point, I am still learning the library, but when I get more comfortable with it I really might submit some guide.

**Comment by feinstein** [comment]
I would also recommend a general guide on regular SQL, since many people (me included) aren't using any ORMs.
